src/omuse/community/pop/interface.f90

```diff
@@ -1139,6 +1187,7 @@ function set_element3d_temperature(i, j, k, temp_, n) result (ret)
   real*8, dimension(n), intent(in) :: temp_
 
   call set_gridded_variable_vector_3D(i, j, k, TRACER(:,:,:,1,curtime,:), temp_, n)
+  call set_gridded_variable_vector_3D(i, j, k, TRACER(:,:,:,1,oldtime,:), temp_, n)
```

Comment:
this is better than not setting oldtime, but may be improved (at a price of adding variables (ie the time derivative)), for the moment ok